src/inference_profiling/concurrent_inference.py

```diff
@@ -0,0 +1,56 @@
+# The given script sends in multiple http requests at a particular time which can be used to test
+# the performance of the torchserve deployed on Cloud run or as a standalone docker container.
+
+# For 10-100 requests, the container memory capacity on cloud run remains upto 50%.
+# CPU performance is around 53% for 10 requests and around 80% for 1000 requests
+# Container instance count 30 for 10 requests and around 139 for 1000 requests
+# Container startup latency 2-3s for 10-1000 requests. Anamolous reading of 5s found once.
+# Max reuest latency was found out to be 1.06 min.Anamolous reading of 7.91 minutes found once.
+
+
+# Importing packages
+import time
+import sys
+import os
+import psutil
+import requests
+import numpy as np
+from futures3.thread import ThreadPoolExecutor
+
+# Defining the model url
+url = "https://crop-mask-example-grxg7bzh2a-uc.a.run.app/predictions/Togo_crop-mask_2019_February"
+
+# Defining the list of images to pass into the inference url
+img_list = np.loadtxt('cloud_img_files.txt', dtype='str')
```

Comment:
1. Where is this txt file?
2. Why not declare it in a python file `src/inference_profiliing/utils.py` and just import it without loading any file?

Reply:
Done!